dll/cpl/sysdm/general.c

```diff
@@ -145,19 +146,23 @@ LRESULT CALLBACK RosImageProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam
                 if (timerid == 0)
                 {
                     HDC hCreditsDC, hLogoDC;
+                    HDC hDC = GetDC(NULL);
                     HFONT hFont;
```

Comment:
Initialize hFont with NULL.

Reply:
This is not the scope of the PR, but I, i take this fix by opportunity